### DISCUSSION TITLE
Second Volume / Partition

### DISCUSSION THREAD

**Discussion by bffive** [grahampugh/erase-install#113] [Q&A]
This script is so handy thank you. I want to run in a student lab were we've set up a separate APFS volume/partition that contains student work. If I use this script using the "--erase" option, will the additional volume/partition remain untouched or will it get wiped and I'll be left with just the Macintosh HD volume/partition?

**Comment by grahampugh** [comment]
Supposedly you can use the `--preservecontainer` option with `startosinstall` to maintain a separate container. I once talked with somebody who had a workflow to move a user folder into a separate container, perform --eraseinstall, then move it back. But I have not tried this, so I don't know the details of how it works.

I have included the `--preservecontainer` with erase-install.sh anyway, so feel free to test it out.
